saleor/dashboard/order/views.py

```diff
@@ -31,19 +31,12 @@
 @staff_member_required
 @permission_required('order.view_order')
 def order_list(request):
-    orders_all = Order.objects.prefetch_related(
-        'groups', 'payments', 'groups__items', 'user')
-    status = request.GET.get('status')
-    if status:
-        orders = orders_all.filter(status=status)
-    else:
-        orders = orders_all.all()
+    orders = (Order.objects.prefetch_related(
+        'groups', 'payments', 'groups__items', 'user').order_by('-pk'))
```

Comment:
When you prefetch `some_field__somefields_field` then `some_field` will be automatically prefetched.

Reply:
Yea, this is not my code but i will change it.

Reply:
I mean, I only added order_by and delete if :)
But this is exactly same code from invoice ;)